category.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * The template for displaying Categories.
+ *
+ *
+ * Learn more: http://codex.wordpress.org/Template_Hierarchy
+ *
+ * @package  WordPress
+ * @subpackage  Timber
+ */
+
+use Timber\Timber;
+
+$templates = [ 'category.twig', 'index.twig' ];
+
+$context              = Timber::get_context();
+$context['page_type'] = get_queried_object();
+$context['wp_title']  = $context['page_type']->name;
+
+wp_register_script( 'load_more', get_template_directory_uri() . '/assets/js/load_more.js', [ 'jquery', 'main' ], '0.0.1', true );
+wp_enqueue_script( 'load_more' );
+
+$post_args = [
+	'posts_per_page' => 10,
```

Comment:
@Wakkatela there is no query argument for the category here
Use the queried object (I think it is WP_Term object in this case also) to query only for posts of the queried category